styles/_article.scss

```diff
@@ -205,14 +205,9 @@
 );
 
   a {
-    display: block;
-    padding: 20px 0;
+    display: inline-block;
+    margin: 20px 0;
```

Comment:
Can we make this rem?

Reply:
No REM in this one, only EMs. I can look at doing that?